### ISSUE TITLE
separate NER loading from NEL loading and retrain NEL model

### ISSUE THREAD

**Issue by kdutia** [TheScienceMuseum/heritage-connector#259]
- [x] figure out whether there is a way to train the linker without having to run the spaCy pipeline (and link candidate retrieval) through the data twice
- [x] get data and label / ask others to label
- [x] rewrite adlib loader and training data so that PERSON labels *lastname, firstname* are swapped to *firstname, lastname*
- [x] correct *lastname, firstname* ordering for old training data
- [x] test new training set using cross-validation
- [x] increase number of link candidates to 15
- [x] clean up smg_loader / datastore so that decisions about loading entities and/or NEL training data from disk are clearer

---
save list of entities produced by `ner_loader.get_list_of_entities_from_es` and `ner_loader.get_link_candidates to disk`?

**Comment by kdutia** [comment]
classifier scores:

``` py
# old
{'SVM': ((0.8844223740826502, 0.08912421163036627),
  (0.7228706235285183, 0.10041834484858267)),
 'logr': ((0.8919374017491464, 0.10628918117728339),
  (0.7648012075643654, 0.06817640123881936)),
 'MLP': ((0.8713725537992454, 0.11534200794704692),
  (0.822890559732665, 0.08127592566564032))}

# new
{'SVM': ((0.8886497819121194, 0.08677406087260922),
  (0.70035237532282, 0.10148084300852585)),
 'logr': ((0.9677087108897876, 0.022773541938428864),
  (0.6802741812642803, 0.09180035154079429)),
 'MLP': ((0.8619334554127696, 0.10147395574544106),
  (0.7831755026412924, 0.10262442318452193))}

# old + new
{'SVM': ((0.8769254388704748, 0.10995371423659453),
  (0.6834557034409567, 0.09570417196836012)),
 'logr': ((0.8547179809590583, 0.120299393935674),
  (0.6775250429553764, 0.09521018633795297)),
 'MLP': ((0.8879791437463588, 0.057258804157695985),
  (0.7760848200865181, 0.07828153530860255))}

# old + new w/ fixed labels
{'SVM': ((0.8951382796803337, 0.08391895944506464),
  (0.6952679947335413, 0.0855428529297813)),
 'logr': ((0.8660420310844714, 0.11971153839969549),
  (0.6815555398216941, 0.08493771696563017)),
 'MLP': ((0.8941988956166027, 0.07761602884634948),
  (0.7714730627908749, 0.0753161934437424))}
```
